react/components/MadieConfirmDialog/index.jsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import PropTypes from "prop-types";
+import ErrorIcon from "@mui/icons-material/Error";
+import MadieDialog from "../MadieDialog";
+
+const MadieConfirmDialog = ({
```

Comment:
Looks like this component is similar to MADiEDeleteDialog and MADiEDiscardDialog. Not sure why we need these many reusable components which are built on reusable dialog, but I see that we could reduce some over head with these dialogs, so I was wondering if we can generalize this MadieConfirmDialog not to be tied to a specific text or specific headers and specific button names, we could make them dynamic and use props to add fill the content ?

Reply:
How about lets not modify MADiEDeleteDialog and MADiEDiscardDialog so no breaking changes, but lets just modify MadieConfirmDialog is be more generic?

Reply:
@RohitKandimalla Sure, that was my thought too. I think the button names, the main message and the warning message are already generic.

Reply:
@RohitKandimalla done